File: C++/Estrutura condicional/Coordenadas.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    double x, y;

    cout << "Valor de X: ";
    cin >> x;
    cout << "Valor de Y: ";
    cin >> y;

    if (x > 0 && y > 0)
    {
        cout << "Q1";
    }
    else if (x < 0 && y > 0)
    {
        cout << "Q2";
    }
    else if (x < 0 && y < 0)
    {
        cout << "Q3";
    }
    else if (x > 0 && y < 0)
    {
        cout << "Q4";
    }
    else if (x == 0 && y == 0)
    {
        cout << "Origem";
    }
    else if (x == 0)
    {
        cout << "Eixo Y";
    }
    else
    {
        cout << "Eixo X";
    }
    
    return 0;
}

File: Python/Estrutura Condicional/Coordenadas.py
x: float; y: float
 
x = float(input("Valor de X: "))
y = float(input("Valor de Y: "))

if (x > 0) and (y > 0):
    print(f"Q1")

elif (x < 0) and (y > 0):
    print(f"Q2")

elif (x < 0) and (y < 0):
    print(f"Q3")

elif (x > 0) and (y < 0):
    print(f"Q4")

elif (x == 0) and (y == 0):
    print(f"Origem")

elif (y == 0) and (x != 0):
    print(f"Eixo X")

else:
    print(f"Eixo Y")